test/test_helper.rb

```diff
@@ -80,6 +80,14 @@ class Comment < ActiveRecord::Base
   belongs_to :commenter
 
   has_many :likes, as: :likeable, dependent: :destroy
+
+  after_commit :foo, on: :destroy
+
+  attr_reader :after_commited
+
+  def foo
```

Comment:
Thanks for the PR @nashby. Could you change the name of this method to something more descriptive. This will provide insight into why we're including this in the `test_helper.rb` down the line.

Something like `set_after_committed_flag` should do.